extensions-contrib/azure-extensions/src/main/java/io/druid/storage/azure/AzureDataSegmentPusher.java

```diff
@@ -139,13 +140,12 @@ public DataSegment uploadDataSegment(
       final long size,
       final File compressedSegmentData,
       final File descriptorFile,
-      final Map<String, String> azurePaths,
-      final boolean replaceExisting
+      final Map<String, String> azurePaths
   )
       throws StorageException, IOException, URISyntaxException
   {
-    azureStorage.uploadBlob(compressedSegmentData, config.getContainer(), azurePaths.get("index"), replaceExisting);
-    azureStorage.uploadBlob(descriptorFile, config.getContainer(), azurePaths.get("descriptor"), replaceExisting);
+    azureStorage.uploadBlob(compressedSegmentData, config.getContainer(), azurePaths.get("index"), true);
+    azureStorage.uploadBlob(descriptorFile, config.getContainer(), azurePaths.get("descriptor"), true);
```

Comment:
It was intentional yes, but now I'm thinking it might be better to just remove it. Honestly, I don't even know for each deep storage implementation how effective 'don't overwrite' semantics that depend on 'if object exists' are, since they are generally eventually consistent (which is why I retained overwrite behavior for most of them). I'll remove.